src/com/chocohead/clm/blocks/MotorBlock.java

```diff
@@ -60,63 +61,78 @@ public String asString() {
 		}
 	}
 
-	public static final DirectionProperty FACING = DirectionProperty.create("facing", Direction.UP, Direction.DOWN);
-	public static final EnumProperty<Status> STATUS = EnumProperty.create("status", Status.class);
+	public enum Rotation implements StringIdentifiable {
+		CLOCKWISE(1),
+		COUNTER_CLOCKWISE(-1);
+
+		public final int multiplier;
+
+		Rotation(int multiplier) {
+			this.multiplier = multiplier;
+		}
+
+		@Override
+		public String asString() {
+			return name().toLowerCase(Locale.ENGLISH);
+		}
+	}
+
+	public static final DirectionProperty FACING = DirectionProperty.of("facing", Direction.UP, Direction.DOWN);
+	public static final EnumProperty<Status> STATUS = EnumProperty.of("status", Status.class);
+	public static final EnumProperty<Rotation> ROTATION = EnumProperty.of("rotation", Rotation.class);
 	private static final VoxelShape UP_SHAPE = VoxelShapes.union(Block.createCuboidShape(4, 0, 4, 12, 13, 12), Block.createCuboidShape(3, 1, 3, 13, 11, 13), Block.createCuboidShape(7, 13, 7, 9, 16, 9));
 	private static final VoxelShape DOWN_SHAPE = VoxelShapes.union(Block.createCuboidShape(4, 3, 4, 12, 16, 12), Block.createCuboidShape(3, 5, 3, 13, 15, 13), Block.createCuboidShape(7, 0, 7, 9, 3, 9));
 
 	public MotorBlock() {
-		super(FabricBlockSettings.copy(Blocks.IRON_BLOCK).breakByTool(FabricItemTags.PICKAXES, 1).build());
+		super(FabricBlockSettings.copyOf(Blocks.IRON_BLOCK).breakByTool(FabricToolTags.PICKAXES, 1));
 
-		setDefaultState(stateFactory.getDefaultState().with(FACING, Direction.UP).with(STATUS, Status.OFF));
+		setDefaultState(stateManager.getDefaultState().with(FACING, Direction.UP).with(STATUS, Status.OFF).with(ROTATION, Rotation.CLOCKWISE));
 	}
 
 	@Override
-	protected void appendProperties(Builder<Block, BlockState> states) {
-		states.with(FACING, STATUS);
+	protected void appendProperties(StateManager.Builder<Block, BlockState> builder) {
+	    builder.add(FACING, STATUS, ROTATION);
 	}
 
 	@Override
+    @Deprecated
```

Comment:
These `@Deprecated` annotations still use spaces, could always remove them completely given Mojang doesn't really mean deprecated

Reply:
I fixed the indentation issue. Inheriting the deprecation suppresses warnings (more effectively than `@SuppressWarnings("deprecation")`), but I can remove it if you prefer that.

Reply:
They're there now so guess we can be "good" and keep them